main/models/models.py

```diff
@@ -418,6 +426,31 @@ def instance_visit_count(self):
     instance_visit_count.fget.short_description = "Nombre de vues"
 
 
+class Section(TimeStampedModel):
```

Comment:
```suggestion
class BaseSection(TimeStampedModel):
```
ça me dérange un peu qu'un peu partout ça soit appelé une `BaseSection` mais que ce soit pas le nom du modèle

Reply:
oui je change ça :)